thavalon-webapp/src/utils/AccountManager.tsx

```diff
@@ -123,7 +123,7 @@ export class AccountManager {
      **/
     private checkRefreshTokenOnTimer(): void {
         const currUnixTime = Math.floor(Date.now() / 1000);
-        const refreshTimeout = (this.expiresAt - currUnixTime) - 898;
+        const refreshTimeout = (this.expiresAt - currUnixTime) - 60;
```

Comment:
Why so much shorter?

Reply:
because before we were checking every 2 seconds, which i didn' mean to commit

Reply:
Oh no, that's a lot of checking